foodsaving/utils/email_utils.py

```diff
@@ -92,107 +92,6 @@ def prepare_emailinvitation_email(invitation):
     }, to=invitation.email)
 
 
-def calculate_group_summary_dates(group):
```

Comment:
I wonder why this shows up as deleted?

Reply:
I did a merge with master, I maybe did the wrong thing. Would be good to check I didn't.